app/controllers/concerns/api_rescuable.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+module ApiRescuable
+  extend ActiveSupport::Concern
+
+  included do
+    rescue_from ActiveRecord::RecordNotFound, with: :handle_record_not_found
+    rescue_from ActiveRecord::RecordInvalid, with: :handle_validation_error
+    rescue_from ActiveRecord::RecordNotUnique, with: :handle_record_not_unique
+    rescue_from ActionController::ParameterMissing, with: :handle_api_error
+    rescue_from Errors::AuthenticateError, with: :handle_unauthenticated
+    rescue_from Errors::Unauthorized, with: :handle_unauthorized
+    rescue_from Errors::Jwt::MissingToken, with: :handle_missing_token
+    rescue_from Errors::Jwt::InvalidToken, with: :handle_invalid_token
+    rescue_from JWT::ExpiredSignature, with: :handle_expired_signature
+  end
+
+  private
+
+    def handle_validation_error(exception)
+      respond_with_error(exception)
+    end
+
+    def handle_record_not_found(exception)
+      respond_with_error(exception.message, :not_found)
```

Comment:
This can be `respond_with_error status: :not_found, message: t("error.record_not_found", model_name: exception.model)`

Reply:
exception.model is always nil. can use exception.message

Reply:
We can also call `id` on this. Eg: `exception.id`.